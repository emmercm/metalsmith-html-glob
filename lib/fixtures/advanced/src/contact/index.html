<!DOCTYPE html>
<html lang="en">
<body>
Contact
<script src="**/*.js"></script>
</body>
</html>
