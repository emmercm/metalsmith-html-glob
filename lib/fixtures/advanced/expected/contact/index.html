<!DOCTYPE html><html lang="en"><head></head><body>
Contact
<script src="/static/js/script.js"></script>


</body></html>